db/migrate/20200205154839_update_vssc_extract_date_value.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateVsscExtractDateValue < ActiveRecord::Migration[5.2]
+    def up
+      DrivetimeBand.update_all("vssc_extract_date='2019-10-31 00:00:00'")
```

Comment:
I like this as part of the migration instead of increasing the complexity of the update code for a one time occurrence 🥇